gtsam/base/Matrix.cpp

```diff
@@ -142,7 +142,7 @@ void print(const Matrix& A, const string &s, ostream& stream) {
   static const Eigen::IOFormat matlab(
       Eigen::StreamPrecision, // precision
       0, // flags
-      " ", // coeffSeparator
+      ", ", // coeffSeparator
```

Comment:
Why? MATLAB does not need it (or do this, I think).

Reply:
Whoops. I put the explanation in the commit message, should have put it here as well.

Separating row values with commas is supported by MATLAB so no effect there,
but this also helps when printing with Python and C++ since the output can be
directly copied without modification.